src/amocrm/client/http.py

```diff
@@ -96,13 +62,18 @@ def get_validated_response(response: Response, url: str, expected_status_codes:
 
 
 def get(url: str, params: dict | None = None, expected_status_codes: list[int] | None = None, cached: bool = False) -> dict[str, Any]:
-    return request(
-        method="get",
-        url=url,
-        params=params,
-        expected_status_codes=expected_status_codes,
-        cached=cached,
-    )
+    def call() -> dict[str, Any]:
```

Comment:
```suggestion
    def fetch() -> dict[str, Any]:
```

`call` в нашем случае не означает ничего, а `fetch` явно даёт понять, что мы ходим в интернет